examples/generate_atmos_dataset.jl

```diff
@@ -4,9 +4,9 @@ using JLD2
 
 time_indices = 1:1
 
-qt = ClimaOcean.JRA55.jra55_field_time_series(:specific_humidity; time_indices)
-Tt = ClimaOcean.JRA55.jra55_field_time_series(:temperature; time_indices)
-pt = ClimaOcean.JRA55.jra55_field_time_series(:sea_level_pressure; time_indices)
+qt = ClimaOcean.JRA55.JRA55_field_time_series(:specific_humidity; time_indices)
+Tt = ClimaOcean.JRA55.JRA55_field_time_series(:temperature; time_indices)
+pt = ClimaOcean.JRA55.JRA55_field_time_series(:sea_level_pressure; time_indices)
```

Comment:
Is this related to this PR?

For changes that are unrelated to a PR just put them in a different PR.

Reply:
hmmm that looks like it's a bugfix. The function name is `JRA55_field_time_series`

Reply:
in the future if you find bugs like this put them in a new PR instead of loading them into an unrelated PR. It will help move things forward faster because it will disentangle PRs.